recipe/build-core.sh

```diff
@@ -13,6 +23,17 @@ build --define CONDA_NM=${NM}
 build --define CONDA_RANLIB=${RANLIB}
 build --define CONDA_SDKROOT=${SDKROOT}
 EOF
+  if [[ "$target_platform" == osx-arm64 ]]; then
+    cat >> .bazelrc <<EOF
+# build --subcommands
+build --crosstool_top=//bazel_toolchain:toolchain
+build --cpu=darwin_arm64
+build --platforms=//bazel_toolchain:target_platform
+build --host_platform=//bazel_toolchain:build_platform
+# build --extra_toolchains=//bazel_toolchain:cc_cf_toolchain
+# build --extra_toolchains=//bazel_toolchain:cc_cf_host_toolchain
```

Comment:
do we need the commented out lines in this?

Also, can we add these lines using `recipe/patches/0006-remove-stdlib-libc-from-.bazelrc.patch` or another patch file?

Reply:
I consider patches as "temporary" fixes for upstream problems. These are more integral to the conda-forge build system than part of the upstream ray build.

Reply:
I removed the commented-out lines here and above.